keanu-python/keanu/context.py

```diff
@@ -6,6 +6,7 @@
 from py4j.java_collections import JavaList, JavaArray, JavaSet, JavaMap
 from typing import Dict, Any, Iterable, List, Collection, Set
 from _io import TextIOWrapper
+from .base import JavaObjectWrapper
```

Comment:
I'm wary of adding this as it's likely to cause us grief with circular dependencies

Reply:
Perhaps the best way to solve this is to just collapse these two files down to like a core.py or something to stop pretending they're not interrelated